src/app/researcher/study/request/[memberSlug]/study-proposal.tsx

```diff
@@ -125,7 +125,7 @@ export const StudyProposal: React.FC<{ memberSlug: string }> = ({ memberSlug })
 
     return (
         <form onSubmit={studyProposalForm.onSubmit((values: StudyProposalFormValues) => createStudy(values))}>
-            <Stack>
+            <Stack pt="lg" pl="xl" pr="xl" pb="xl" gap="sm">
```

Comment:
if we really need the padding here ideally we could get away with a simple `p='xl'` to clean it up